testutils/src/main/scala/za/co/absa/enceladus/testutils/datasetComparison/Flatten.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.datasetComparison
+
+import org.apache.log4j.{LogManager, Logger}
+import org.apache.spark.sql.{Column, DataFrame}
+import org.apache.spark.sql.functions.{expr, max}
+import org.apache.spark.sql.types._
+
+import scala.collection.mutable
+
+object Flatten {
```

Comment:
I would probably call this `FlattenFuncitons` or something like that, maybe throw `helper` in the mix? Just a suggestion.

Reply:
Done. I think.